streams/src/main/java/org/apache/kafka/streams/kstream/internals/KStreamImpl.java

```diff
@@ -584,32 +651,38 @@ public void process(final ProcessorSupplier<? super K, ? super V> processorSuppl
 
         joinThis.ensureJoinableWith(joinOther);
 
-        return join.join(joinThis,
-                         joinOther,
-                         joiner,
-                         windows,
-                         joined);
+        return join.join(
+            joinThis,
+            joinOther,
+            joiner,
+            windows,
+            joined
+        );
     }
 
     /**
      * Repartition a stream. This is required on join operations occurring after
      * an operation that changes the key, i.e, selectKey, map(..), flatMap(..).
      *
      * @param keySerde Serdes for serializing the keys
-     * @param valSerde Serdes for serilaizing the values
+     * @param valSerde Serdes for serializing the values
      * @return a new {@link KStreamImpl}
      */
     private KStreamImpl<K, V> repartitionForJoin(final Serde<K> keySerde,
                                                  final Serde<V> valSerde) {
-        OptimizableRepartitionNode.OptimizableRepartitionNodeBuilder repartitionNodeBuilder = OptimizableRepartitionNode.optimizableRepartitionNodeBuilder();
-        String repartitionedSourceName = createRepartitionedSource(builder,
-                                                                   keySerde,
-                                                                   valSerde,
-                                                                   null,
-                                                                   name,
-                                                                   repartitionNodeBuilder);
-
-        OptimizableRepartitionNode<K, V> repartitionNode = repartitionNodeBuilder.build();
+        final OptimizableRepartitionNode.OptimizableRepartitionNodeBuilder<K, V> repartitionNodeBuilder =
+            OptimizableRepartitionNode.optimizableRepartitionNodeBuilder();
+
+        final String repartitionedSourceName = createRepartitionedSource(
```

Comment:
This is a meta question: Is this re-alignment with the first parameter in new line necessary for removing the warning?